projects/aca-content/src/lib/components/details/details.component.ts

```diff
@@ -99,6 +99,22 @@ export class DetailsComponent extends PageComponent implements OnInit, OnDestroy
       .subscribe((aspectActions) => {
         this.aspectActions = aspectActions;
       });
+    this.infoDrawerOpened$
+      .pipe(
+        filter((opened) => !opened),
+        first(),
```

Comment:
Same here, you can use only `first` to filter

Reply:
done